### PR TITLE
fix: add database configuration in docker runining

### PR DESCRIPTION
**Pull request by zcy530**
Upload readme.md

### CODE REVIEW

**Comment by hsluoyz** [comment]
@ComradeProgrammer plz review

File: web/src/Conf.js [hunk 1]
```diff
@@ -12,7 +12,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-export const ShowGithubCorner = false;
```
Comment (sagilio) [inline web/src/Conf.js:15]: These changes seem unrelated to this PR

File: main.go [hunk 2]
```diff
@@ -29,7 +29,7 @@ import (
 )
 
 func main() {
-	createDatabase := flag.Bool("createDatabase", false, "true if you need casdoor to create database")
+	createDatabase := flag.Bool("createDatabase", true, "true if you need casdoor to create database")
```
Comment (sagilio) [inline main.go:32]: Why not add this to conf or env？

File: README.md [hunk 3]
```diff
@@ -179,8 +206,6 @@ And undeploy it with:
 make undeploy
 ```
 
-That's it! Try to visit http://localhost:8000/. :small_airplane:
```
Comment (sagilio) [inline README.md:182]: Why move this ?

File: README.md [hunk 4]
```diff
@@ -160,11 +166,32 @@ dataSourceName = root:123456@tcp(db:3306)/
...
+
+
+
+Unlike the casdoor-all-in-one image, this image don't have a MySql database inside, which means you need to create a database named `casdoor`, before running docker.
+
+
+```bash
+
+# into database container
+docker exec -it casdoor_db_1 bash
```
Comment (sagilio) [inline README.md:179]: Container names may be different for other users, and I think most developers should know how to use the database, we need not descript this.

**Review by sagilio — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (zcy530) [reply README.md:182]: I think it is possible to visit http://localhost:8000/ before deployment,  so I moved it behind docker-compose up

File: README.md (hunk 4 above)
Comment (zcy530) [reply README.md:179]: I am trying to solve the issue raised by this friend. And thanks for reminding me, I found the container with this name is inside the image, so I thought it was the same for everyone?
https://github.com/casbin/casnode/issues/445

Comment (sagilio) [reply README.md:179]: If the user does not edit the `docker-compose.yml` file, the name will indeed be this.
But if someone uses `docker stack` to run the `docker-compose.yml file, it may be another name.
I still think we only need to notify the user about the option of auto-create DB default is false.

File: README.md (hunk 3 above)
Comment (sagilio) [reply README.md:182]: Ok, but description about k8s will lack this (:

**Comment by hsluoyz** [comment]
This PR lacks a lot of things. I think it's not a good base for us to work on.

File: main.go (hunk 2 above)
Comment (RyaoChengfeng) [reply main.go:32]: @sagilio Why not automatically create the database if it doesn't exist, but configure it here? I think the auto-created behavior would be better.
